Assignment/Program.cs

```diff
@@ -1,7 +1,59 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+﻿namespace Training;
+class Spark {
+   public static void Main () {
+      int moneyGiven = 0;
+      int chocolatePrice = 0;
+      int wrapperValue = 0;
+      GetInput (ref moneyGiven, ref chocolatePrice, ref wrapperValue);
+      Console.WriteLine ();
+      GetResult (moneyGiven, chocolatePrice, wrapperValue);
+   }
+
+   /// <summary>
+   /// Gets input from user
+   /// </summary>
+   /// <param name="moneyGiven"></param>

```

Comment:
Summary tag can be in a single line. When naming param you have describe the usage of it.

Reply:
Apply param fix